source/prefill_payment_links/index.html.md.erb

```diff
@@ -0,0 +1,40 @@
+---
+title: Prefill payment reference and amount when using a payment link
+last_reviewed_on: 2022-07-21
+review_in: 6 months
+weight: 104
+---
+
+# Prefill payment reference and amount when using a payment link
+
+You can prefill the payment reference and amount when you send a payment link to your users. When the user opens your payment link, the **Total to pay** and reference fields will already be completed.
```

Comment:
Do we want to bold both `total to pay` and `reference`?
Just wondering why just one item is in bold?

Reply:
Good spot! The reason for that is because services can name the reference field, so we don't know what that field will be called. Total to pay is the same for every service, though, as far as I'm aware.

Reply:
Sound good - will approve now.